packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -1645,3 +1656,15 @@ cp1Rusd_rseliniParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_rseliniParentConfig_priceHaircutUnscaled = "0"
 cp1Rusd_rseliniParentConfig_autoExchangeDiscountUnscaled = "0"
 cp1Rusd_rseliniParentConfig_oracleNodeId = "<%= settings.seliniUsdcReyaLmNodeId %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_ramber_capUnscaled = "<%= formatEther(MaxUint256) %>"
+cp1Rusd_ramber_autoExchangeThresholdUnscaled = "0"
```

Comment:
hmm, actually let's flag in the team chat that we will need to make an explicit decision on the haircuts / discounts that we want to use for the ramber and rselini token, there're a few neuance s within this that will be good to discuss

one thing worth noting is that the effectively liquidity / max exposure the pool can support in perps trading will decrease as a result of introducing ramber and rselini if we have haircuts of zero